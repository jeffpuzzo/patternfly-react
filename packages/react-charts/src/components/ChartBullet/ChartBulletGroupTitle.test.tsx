import * as React from 'react';
import { render } from '@testing-library/react';
import { ChartBulletGroupTitle } from './ChartBulletGroupTitle';

Object.values([true, false]).forEach(() => {
  test('ChartBulletGroupTitle', () => {
    const view = render(<ChartBulletGroupTitle />);
    expect(view.container.outerHTML).toMatchSnapshot();
  });
});

test('renders component data', () => {
  const view = render(<ChartBulletGroupTitle title="Text label" subTitle="Measure details" />);
  expect(view.container.outerHTML).toMatchSnapshot();
});
