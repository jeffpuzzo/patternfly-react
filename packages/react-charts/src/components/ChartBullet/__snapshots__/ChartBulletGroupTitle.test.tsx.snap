// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartBulletGroupTitle 1`] = `"<div><div style=\\"pointer-events: none; position: relative; width: 100%; height: 100%;\\" class=\\"pf-c-chart\\"><svg width=\\"450\\" height=\\"300\\" role=\\"img\\" viewBox=\\"0 0 450 300\\" style=\\"pointer-events: all; width: 100%; height: 100%;\\"></svg><div style=\\"z-index: 99; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;\\"><svg width=\\"450\\" height=\\"300\\" viewBox=\\"0 0 450 300\\" style=\\"overflow: visible; width: 100%; height: 100%;\\"></svg></div></div></div>"`;

exports[`ChartBulletGroupTitle 2`] = `"<div><div style=\\"pointer-events: none; position: relative; width: 100%; height: 100%;\\" class=\\"pf-c-chart\\"><svg width=\\"450\\" height=\\"300\\" role=\\"img\\" viewBox=\\"0 0 450 300\\" style=\\"pointer-events: all; width: 100%; height: 100%;\\"></svg><div style=\\"z-index: 99; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;\\"><svg width=\\"450\\" height=\\"300\\" viewBox=\\"0 0 450 300\\" style=\\"overflow: visible; width: 100%; height: 100%;\\"></svg></div></div></div>"`;

exports[`renders component data 1`] = `"<div><div style=\\"pointer-events: none; position: relative; width: 100%; height: 100%;\\" class=\\"pf-c-chart\\"><svg width=\\"450\\" height=\\"300\\" role=\\"img\\" viewBox=\\"0 0 450 300\\" style=\\"pointer-events: all; width: 100%; height: 100%;\\"><text direction=\\"inherit\\" dx=\\"0\\" x=\\"225\\" y=\\"48.675000000000004\\"><tspan x=\\"225\\" dx=\\"0\\" dy=\\"0\\" text-anchor=\\"middle\\" style=\\"font-size: 24px; text-anchor: middle; font-family: var(--pf-chart-global--FontFamily); letter-spacing: var(--pf-chart-global--letter-spacing); fill: #252525; stroke: transparent;\\">Text label</tspan><tspan x=\\"225\\" dx=\\"0\\" dy=\\"13.5\\" text-anchor=\\"middle\\" style=\\"fill: #b8bbbe; font-size: 14px; text-anchor: middle; font-family: var(--pf-chart-global--FontFamily); letter-spacing: var(--pf-chart-global--letter-spacing); stroke: transparent;\\">Measure details</tspan></text><line vector-effect=\\"non-scaling-stroke\\" x1=\\"0\\" x2=\\"450\\" y1=\\"103.81\\" y2=\\"103.81\\" style=\\"fill: #f0f0f0; opacity: 1; stroke: #f0f0f0; stroke-width: 2;\\"></line></svg><div style=\\"z-index: 99; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;\\"><svg width=\\"450\\" height=\\"300\\" viewBox=\\"0 0 450 300\\" style=\\"overflow: visible; width: 100%; height: 100%;\\"></svg></div></div></div>"`;
